config/dev/azure-credentials.yaml

```diff
@@ -23,3 +23,15 @@ metadata:
 stringData:
   clientSecret: "${AZURE_CLIENT_SECRET}"
 type: Opaque
+---
+apiVersion: hmc.mirantis.com/v1alpha1
+kind: Credential
+metadata:
+  name: azure-cluster-identity-cred
+  namespace: ${NAMESPACE}
+spec:
+  identityRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1beta2
```

Comment:
Wrong API version. Should be `infrastructure.cluster.x-k8s.io/v1beta1`

Reply:
Fixed, thanks!